docs/Earn/bridge/bridge-introduction.md

```diff
@@ -1,6 +1,59 @@
 ---
-title: Introduction
+title: Ankr Bridge
 id: bridge-introduction
 ---
+# Ankr Bridge
+
+## What is Ankr Bridge?
+
+Ankr Bridge is an Ankr product for bridging assets between networks. 
+
+It allows you to transfer your Ankr Liquid Staking tokens, obtained when staking your original assets, to a different network where they remain a valid stake and continue to accumulate rewards.
```

Comment:
suggest brackets around (obtained when staking your original assets). Long sentence.

Reply:
noted.

It allows you to transfer your Ankr Liquid Staking tokens to a different network where they remain a valid stake and continue to accumulate rewards.

You receive Liquid Staking tokens when staking your original assets, in 1:1 ratio.